src/luarocks/fs/win32/tools.lua

```diff
@@ -204,11 +204,17 @@ end
 --- Helper function for fs.set_permissions
 -- @return table: an array of all system users
 local function get_system_users()
+   local exclude = {
+      [""]              = true,
+      ["Name"]          = true,
+      ["\128\164\172\168\173\168\225\226\224\160\226\174\224"] = true, -- Administrator in cp866
```

Comment:
do we potentially need to add other locales in here?
and what if the person has that username but is not in codepage 866?

Reply:
> do we potentially need to add other locales in here?

I think yes if you whant to exclude administrator user. Windows creates such users by default.

> and what if the person has that username but is not in codepage 866?